README.md

```diff
@@ -493,6 +521,49 @@ codePush.checkForUpdate()
 });
 ```
 
+#### codePush.disallowRestart
+
+```javascript
+codePush.disallowRestart(): void;
+```
+
+Temporarily disallows programmatic restarts to occur as a result of either of following scenarios:
+
+1. A CodePush update is installed using `InstallMode.IMMEDIATE`
```

Comment:
why is this numbered list not separated by new lines while the one in `allowRestart` is?